src/Tribe/Query.php

```diff
@@ -68,6 +68,11 @@ public static function can_inject_date_field( $query ) {
 		 * @param WP_Query $query
 		 */
 		public static function parse_query( $query ) {
+			// If this is set then the class will bail out of any filtering.
+			if ( $query->get( 'tribe_suppress_filters', false ) ) {
```

Comment:
Thought: Perhaps this should be `tribe_suppress_query_filters`, maybe? "Suppress filters" seems a bit too generic.